### PR TITLE
Run latexmk ci on PRs.

### PR DESCRIPTION
**Pull request by yjyao**

### CODE REVIEW

**Comment by passamo9** [comment]
Good lord this is going over my head! 😂

Can I be of any assistance?

**Comment by yjyao** [comment]
> Good lord this is going over my head! 😂

sorry. i thought the title was clear but apparently not. some context:

i found out that currently the latexmk ci (the ci that builds/uploads example PDFs) only runs when an actual commit is being made to your repo (in most cases, when you merge our PRs). but we want to check that the PDFs compile in PRs as well. this PR fixes that as you can see at the bottom of the page:
image

File: .github/workflows/latexmk.yml [hunk 1]
```diff
@@ -15,7 +15,8 @@ jobs:
             examples
           latexmk_use_xelatex: true
       - uses: fregante/setup-git-user@v1
-      - name: Upload PDF to examples branch
+      - if: github.event_name == 'push'
```
Comment (yjyao) [inline .github/workflows/latexmk.yml:18]: this makes sure that the example branch is not updated by PRs (in progress), and only updated when changes are pushed to the repo.

**Comment by passamo9** [comment]
That is so much clearer for me, thank you!!

File: .github/workflows/latexmk.yml (hunk 1 above)
Comment (passamo9) [reply .github/workflows/latexmk.yml:18]: Good to know!

Please let me know if you would like this to be merged

Thanks again!

**Comment by yjyao** [comment]
i think this is ready to go. unless you would like to get someone else to review/approve first, which is good practice too.

**Comment by passamo9** [comment]
@yjyao That's a great idea.

Does this need to be an issue, or would you like to summon someone with the @?

**Review by bond7o6 — APPROVED** [review]
Looks good

**Pull request merged**